Improve documentation (#1019)
Automattic/wordpress-activitypub: docs/translations.md

---
title: Translate
layout: page
nav_order: 2
description: "How To Translate the ActivityPub Plugin"
permalink: /translate/
---

# How To Translate the ActivityPub Plugin

## Who translates the ActivityPub plugin? How can I get involved?

Anyone can help by suggesting new translations [here](https://translate.wordpress.org/projects/wp-plugins/activitypub/).

Once you've suggested a new translation, [a GlotPress validator](https://make.wordpress.org/polyglots/teams/) will review it. They will then approve, reject, or change your suggestions. Once your changes get approved, they are automatically shipped to all ActivityPub plugin users using WordPress in your language.

## How does GlotPress work?

> You can find detailed GlotPress documentation on [this page](https://make.wordpress.org/polyglots/handbook/tools/glotpress-translate-wordpress-org/).

Reviewer: ```suggestion
You can find detailed GlotPress documentation in [the WordPress Polyglots Handbook](https://make.wordpress.org/polyglots/handbook/tools/glotpress-translate-wordpress-org/).
```

## I want to change translations locally. Where can I download the .PO file for my language?

You'll find options to create a .PO file and export translations at the bottom of each language page on translate.wordpress.org.

> ## I found a missing translation, but I can’t find the string in GlotPress

Reviewer: ```suggestion
## I found a missing translation, but I can’t find the string in GlotPress.
```
